azure-stack-go-sdk/azure-stack-go-sdk-profiles.md

```diff
@@ -0,0 +1,302 @@
+```
+
+# Install Azure Go SDK on Azure Stack
+1. Follow the official instructions to install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+2. Follow the official instructions to install [Go](https://golang.org/dl/). Note that to use profiles you need to install Go version 1.9 or newer.
+3. Install Go SDK and its dependencies by running the following bash command:
+
+    ```bash
+    go get -u -d github.com/Azure/azure-sdk-for-go/...
+    ```
+    * Full details for Azure Go SDK Installation are [here](https://docs.microsoft.com/en-us/go/azure/azure-sdk-go-install#get-the-azure-sdk-for-go)
+    * Azure Go SDK is publicly available at [Go SDK GitHub](https://github.com/Azure/azure-sdk-for-go)
+    * Azure Go SDK uses Go-AutoRest modules to send requests to Azure resource provider endpoints. To use Azure services in code, in addition to Azure Go SDK, modules must be imported from [Go-AutoRest](https://github.com/Azure/go-autorest). More details about the most common Go-Autorest packages can be found [here](https://docs.microsoft.com/en-us/go/azure/azure-sdk-go-install#including-the-azure-sdk-for-go-in-your-project). For a complete list of the available modules in Go-AutoRest go [here](https://godoc.org/github.com/Azure/go-autorest).
+
+# How to use GO SDK profiles on Azure Stack
+To run a sample of Go code on Azure Stack:
+
+1. Install Azure Go SDK and its dependencies (see previous section)
+2. Get metadata information from resource manager endpoint. Resource manager metadata endpoint returns a JSON file with required information to run Go samples. To retreive the metadata, open a web browser in your Azure Stack environment and use the url: `ResourceManagerURL`/metadata/endpoints?api-version=1.0
+
+The `ResourceManagerURL` varies based on the region name, machine name and external fully qualified domain name (FQDN) of your Azure Stack deployment: 
+
+|Environment|ResourceManagerURL|
+|-----------|------------------|
+|Development kit|https://management.local.azurestack.external/|
+|Integrated systems|https://management.(region).ext-(machine-name).(FQDN)|
+    
+Sample JSON file from a development kit environment (https://management.local.azurestack.external/metadata/endpoints?api-version=1.0): 
+
+```json
+    {
+        "galleryEndpoint": "https://portal.local.azurestack.external:30015/",
+        "graphEndpoint": "https://graph.windows.net/",
+        "portalEndpoint": "https://portal.local.azurestack.external/",
+        "authentication": {
+            "loginEndpoint": "https://login.windows.net/",
+            "audiences": [
+                "https://management.azurestackiXX.onmicrosoft.com/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+            ]
+        }
+    }
+```
+3. If not available, create a subscription and save the subscription ID to be used later. Instructions to create a subscription are [here](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-subscribe-plan-provision-vm)
```

Comment:
Never link to another document with `here`. Use the full name of the document in this style: `For instructions, see [article name].`